main.py

```diff
@@ -0,0 +1,11 @@
+# Simple Python Program to determine whether a given integer is Prime : 
+integer = 9
+if integer > 1:
+    for i in range (2,(integer//2)+1):
```

Comment:
Add a comment above the loop explaining what the loop does.

Reply:
Added comment above the loop